### ISSUE TITLE
Getting Render Error

### ISSUE THREAD

**Issue by rassemdev** [computerjazz/react-native-draggable-flatlist#330]
When I am using ScaleDecorator from your package, i am getting this error.

ERROR  Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `RowItem.

I am using Functional component. and my package details is:
    "react": "17.0.1",
    "react-native": "0.64.2",
    "react-native-draggable-flatlist": "^2.6.2",
    "react-native-gesture-handler": "^1.10.3",
    "react-native-reanimated": "^2.2.0",

**Comment by chungweileong94** [comment]
@rassemdev The README is misleading for sure. And the current documentation is actually for version `3.0.0-beta.13`.

https://www.npmjs.com/package/react-native-draggable-flatlist?activeTab=versions

**Comment by computerjazz** [comment]
thanks @chungweileong94. I'm thinking about releasing `3.0.0` soon but am looking for a few volunteers do some stress-testing beforehand. Have you had a chance to use it?

**Comment by chungweileong94** [comment]
> thanks @chungweileong94, I added a link in the README for version 2.
> 
> 
> 
> I'm thinking about releasing `3.0.0` soon but am looking for a few volunteers do some stress-testing beforehand. Have you had a chance to use it? 

Yeah definitely. I did use it recently, looks good to me, quite solid.

**Comment by rassemdev** [comment]
> @rassemdev The README is misleading for sure. And the current documentation is actually for version `3.0.0-beta.13`.
> 
> https://www.npmjs.com/package/react-native-draggable-flatlist?activeTab=versions

Yes, I used that one too. But too much lag. Not even moving smoothly.

**Comment by computerjazz** [comment]
@rassemdev would it be possible to post a repro snack of the lagginess so I can try to fix it?

**Comment by rassemdev** [comment]
> @rassemdev would it be possible to post a repro snack of the lagginess so I can try to fix it?

I tried one of my client project, which is confidential. But i will try to create a new one to see if the lagging is available or not.

**Comment by rassemdev** [comment]
I created a new repo and install this package. it is working good, and also after tweaking my old project it was working again. Still alot of area need to improve.

**Comment by baraaAljabali** [comment]
I don't understan why this is closed? I'm getting this error using the new documentation.. how cn it be fixed?

**Comment by chungweileong94** [comment]
> I don't understan why this is closed? I'm getting this error using the new documentation.. how cn it be fixed?

Yes, this issue has already been fixed, as the stable v3 is released. 
So if you are using v2, you might need to refer to this [documentation](https://github.com/computerjazz/react-native-draggable-flatlist/blob/b60c23035fc0371a735cde1d000e9ad7f8e0f049/README.md) instead, but do note that v2 doesn't have "Cell Decorators".
